behat/features/bootstrap/FeatureContext.php

```diff
@@ -508,4 +510,23 @@ public function theCiBuildItemForBuildShouldBeCreated($title) {
       throw new \Exception($error_message);
     }
   }
+
+  /**
+   * @BeforeSuite
+   *
+   * Set variable 'test_mode' to TRUE to bypass some checks.
```

Comment:
better expand on the "some checks", e.g. `to prevent asserting .shoov.yml exists on the repository.`